week1/homework/src/server.js

```diff
@@ -10,10 +10,35 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    const requestUrl = request.url;
+    switch (requestUrl) {
+      case '/state':
+      response.statusCode = 200;
```

Comment:
This will be applied only on this case, so the following cases (except for the error case - default) will take the default status-code, which is luckily 200.